pkg/core/tekton/tekton.go

```diff
@@ -210,6 +242,8 @@ func generatePipeline(w workflow.Workflow, namespace string) *v1beta1.Pipeline {
 			resolver.addReference(fmt.Sprintf("inputs.%s.name", *input.Name), fmt.Sprintf("$(params.%s)", codesetNameParam))
 			pb.ParamWithDefaultValue(codesetVersionParam, "Codeset version (git revision)", "main")
 			resolver.addReference(fmt.Sprintf("inputs.%s.version", *input.Name), fmt.Sprintf("$(params.%s)", codesetVersionParam))
+			pb.Param(codesetProjectParam, "Reference to the codeset project (git organization)")
```

Comment:
Why is this part of the `workflow run list` implementation ? and more importantly, why did the pipeline work before without this parameter ? Are you using it just to have a git org reference in your pipeline run, which then you can use to filter workflow runs by project name ?

Reply:
Nevermind, I just got my answer. All I had to do is look at the commit msg :)